runelite-client/src/main/java/net/runelite/client/plugins/timetracking/farming/FarmingTracker.java

```diff
@@ -187,10 +190,16 @@ public PatchPrediction predictPatch(FarmingPatch patch)
 
 		PatchState state = patch.getImplementation().forVarbitValue(value);
 
+
 		int stage = state.getStage();
 		int stages = state.getStages();
 		int tickrate = state.getTickRate() * 60;
 
+		if (config.diseasedCropNotification() && state.getCropState() == CropState.DISEASED)
+		{
+			notifier.notify("You have a diseased crop, cure it quick!");
```

Comment:
This should probably have more information, like crop type and location.

Reply:
Added location and crop type.